web/main.rb

```diff
@@ -4,31 +4,38 @@
 require 'string-utility'
 require_relative '../utils/utilities'
 
-gh = Utilities.configure_stuff(token: ENV['GHULS_TOKEN'])
-demonyms = YAML.load_file("#{Dir.pwd}/web/public/demonyms.yml")
-adjective_path = "#{Dir.pwd}/web/public/adjectives.txt"
+$gh = Utilities.configure_stuff(token: ENV['GHULS_TOKEN'])
+$demonyms = YAML.load_file("#{Dir.pwd}/web/public/demonyms.yml")
+$adjective_path = "#{Dir.pwd}/web/public/adjectives.txt"
```

Comment:
You should be able to access them from the analyze method as the variable declarations are in the scope above the analyze method. Could you give it a run/try with the global declarations removed?